backend/production-optimiser-api/src/main/java/it/polimi/productionoptimiserapi/dtos/AccountRequestDTO.java

```diff
@@ -0,0 +1,26 @@
+package it.polimi.productionoptimiserapi.dtos;
+
+import it.polimi.productionoptimiserapi.entities.AccountRequest;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class AccountRequestDTO {
+
+  private String id;
+
+  @NotNull
+  @Size(min = 1, max = 50)
+  @Email
+  private String email;
+
+  @NotNull private String message;
+
```

Comment:
add status field with enum APPROVED, DENIED
that way on ticket resolution we don't delete entities from db, just change their status

can be null, return it to frontend with status so it can be shown but on incomming requests its ignored/not filled

Reply:
do we need to keep requests once they have been processed though?